app/views/compute_resources_vms/form/vmware/_network.html.erb

```diff
@@ -1,9 +1,9 @@
 <%= select_f f, :type, compute_resource.nictypes, :first, :last, { },
     :class => "col-md-3 vmware_type",
-    :size => "col-md-8", :disabled => !new_vm,
```

Comment:
We shouldn't remove this, since we're not allowing to update the `compute_attributes` for existing VMs.